configure_kernels.py

```diff
@@ -0,0 +1,39 @@
+"""
+Writes kernel.json and terminal .bash_profile environment variables
```

Comment:
In order to tidy things up even a bit more, I would define two helper functions in this module, one for the kernels configuration (which corresponds to lines 14-31) and one for the terminal configuration (33-39 plus the setting of the alias for python, see my other comment). And I would call them from:
```python
if __name__ == '__main__':
    configure_kernels()
    configure_terminal()
```
Also the name of this new Python file should be `configure_kernels_and_terminal.py`.

Reply:
done,